packages/web/src/djitsu/style.less

```diff
@@ -6,4 +6,14 @@
 
 .main {
   background: green;
+}
+
+// global style overrides
+.djs-theme {
```

Comment:
Not the relevant file

If we're customizing the notification - I'd suggest creating a custom component that will be used in-place of the existing uses of the notification, and associating the style change in that component.

Additionally in the project we're not writing any CSS to the less files, unless there's a special case/need. All of the component styling is handled via `styled-components`